examples/triangle_xform/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "triangle_xform"
+version = "0.1.0"
+authors = ["Gonzalo Brito Gadeschi <gonzalobg88@gmail.com>"]
```

Comment:
You should put your name here, I didn't do anything :D

Reply:
Nah, you've done the hard work on the packed_simd stuff. Is it OK to add something like "the packed_simd developers"?

Reply:
We should add "The Rust project developers..." copyright notice to all files, since the plan is to put this in std at some point to be able to use this in stable because we don't want to stabilize `repr(simd)`, the llvm intrinsics, etc.

Reply:
Also, you are doing a lot of hard work too!